PyExpLabSys/drivers/vaisala_dmt143.py

```diff
@@ -75,13 +83,15 @@ def water_level(self):
         The actual measurements from the device.
         """
         command = 'SEND'
-        raw_value = self.comm(command)
-        # One could consider to use the FORMAT command
-        # to make the output less cryptic...
-        split_values = raw_value.split(' ')
-        dew_point = float(split_values[2])
-        dew_point_atm = float(split_values[6])
-        vol_conc = float(split_values[10])
+        raw_value = self.comm(command, single_line=True)
+        if raw_value is None:
+            return
+        split_values = raw_value.split(';')
+        # print(split_values)
+
+        dew_point = float(split_values[0])
+        dew_point_atm = float(split_values[1])
+        vol_conc = float(split_values[2])
```

Comment:
Same here with list comprehension and unpacking

Reply:
See above